### ISSUE TITLE
yum install command in Dockerfile can be split up

### ISSUE THREAD

**Issue by gregcorbett** [apel/rest#56]
Previous versions of the base CentOS images (from which our images are based) had issues around the `overlayfs` backend used when building new images. The workaround at the time was to only run yum once.

According to [Overlayfs and yum](https://hub.docker.com/_/centos/) the more recent base CentOS images have the required `yum-plugin-ovl` installed and enabled, fixing the problems that led to the workaround being applied originally. 

As a result, the large yum command in the [Dockerfile](../blob/1.5.0-1/Dockerfile#L25) could be broken down into smaller yum commands for clarity sake. It may also speed up `docker pull` commands by having more, smaller, layers in the image.

**Comment by tofu-rocketry** [comment]
Was this an ordering problem originally?

**Comment by gregcorbett** [comment]
I don't think so, it was a yum checksum issue with any two successive yum installs.
